optaplanner-core/src/main/java/org/optaplanner/core/api/score/AbstractScore.java

```diff
@@ -34,6 +35,74 @@
 
     protected static final String INIT_LABEL = "init";
 
+    protected static int divideScoreLevel(int score, int scoreDivisor) {
+        return divideScoreLevel(score, scoreDivisor, null);
+    }
+
+    protected static int divideScoreLevel(int score, int scoreDivisor, String maybeLabel) {
+        String label = maybeLabel == null ? "" : maybeLabel;
```

Comment:
Nitpicking: You have just moved one instance of `SimpleScore`'s label to another place. What I was pointing out is that `""` is the score label of `SimpleScore` and I noticed the the string literal (`""`) is used multiple times in the `SimpleScore` class and therefore should be replaced by a constant (provided that all `""` occurrences have the same meaning, which I presume they do).

In another words, what would happen if we decided that `SimpleScore` will get a new label of `simp`? We would replace two `""` occurrences in `SimpleScore` with `"simp"` and we would probably miss this instance (on line 43).

Reply:
What would happen is that the exception message would continue to say "X" instead of "Xsimp". 
I really don't consider this a problem - it was a problem before, where I was propagating something that should have been a constant. I don't anymore.

Reply:
OK, that's another, valid point of view that I didn't think of.